slam-ws/src/nav_goal/bin/movebase_client

```diff
@@ -5,17 +5,16 @@ import rospy
 import sys
 
 if __name__ == '__main__':
-   if len(sys.argv) < 4:
-        rospy.logerr("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
-   else:
-	try:
-		x = float(sys.argv[1])
-		y = float(sys.argv[2])
-		yaw = float(sys.argv[3])
+   	if len(sys.argv) < 4:
+		   rospy.logerr("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
+   	else:
+		try:
+			x = float(sys.argv[1])
+			y = float(sys.argv[2])
+			yaw = float(sys.argv[3])
```

Comment:
Now here's an issue of tabs vs spaces. Configure your VSCode to use 1 tab == 4 spaces so we can be consistent and keep the code base clean

Reply:
Sounds good, done